data-access-request-faq.md

```diff
@@ -0,0 +1,52 @@
+
+
+## Google Cloud Platform Questions
+
+### Why can't I sign up with my Gmail account?
+The License Agreement is signed by your institution (the 'Legal Entity'), which means that your institution will be liable if the License Agreement is breached. As your institution gives you a personal institutional email address, they also manage the access to this account. By using your institutional email address, we have the guarantee that you are acting as a representative of your institution and that your access will be revoked in case of, for example, termination of your contract.
+
+### I can't seem to set up a GCP account with my institutional email address?
+Please follow the instructions in our [Getting a Google account](https://hartwigmedical.github.io/documentation/getting-a-Google-account.html) documentation. In short, go to the [Google Account creation page](https://accounts.google.com/signup?hl=en), click 'Use my current email address instead' and follow the prompts. Please note that we need multi-factor authentication enabled for all accounts we share our data with.
+
+### Can you help us with setting up a GCP account?
+We are not able to set the GCP account up for you. Instructions on getting a GCP account can be found in our [Getting a Google account](https://hartwigmedical.github.io/documentation/getting-a-Google-account.html) instructions. If the instructions are somehow not working for you, please contact us. We might be able to point you to someone that can assist you in the process.
+
+
+## Billing Questions
+
+### Will I be billed for accessing your data?
+We don't personally bill you for accessing our data, but accessing the data through Google Cloud Platform does come with costs. Basically, there are two ways to approach the data:
+- **Download the data onto your own servers**. This will result in egress costs (data that is pulled over the network will be charged per GB, see [Google's Network Pricing page](https://cloud.google.com/compute/network-pricing) for more details.
+- **Analyse the data on GCP itself**. This will results in storage and compute costs. More information can be found on [Google's Cloud Storage pricing page](https://cloud.google.com/storage/pricing) and [Google's VM Instances pricing page](https://cloud.google.com/compute/vm-instance-pricing). If you analyse the data in the same region as it is stored, you won't be charged egress costs. 
+
+What option to choose depends on your use case and specific requirements. 
+
+### How much will I get billed for my download?
+A resource that might help you in estimating costs is the [Google Pricing calculator](https://cloud.google.com/products/calculator).
```

Comment:
I'm not sure if the calculator has network costs, good to link the network pricing page here.

Reply:
```suggestion
A resource that might help you in estimating costs is the [Google Pricing calculator](https://cloud.google.com/products/calculator). Another factor that you need to include when estimating cost, is the charge that you will get for [egress cost](https://cloud.google.com/compute/network-pricing).
```